src/app/api/og/photographer/route.tsx

```
@@ -41,34 +30,133 @@ export async function GET(request: Request) {
-        <div style="display: flex; flex-direction: column;">
-          <div style="font-size: 48px; font-weight: 700; color: white; line-height: 1.2; margin-bottom: 8px;">${name}</div>
-          ${company ? `<div style="font-size: 24px; color: rgba(255,255,255,0.7); margin-bottom: 12px;">${company}</div>` : ""}
-          ${location ? `<div style="display: flex; align-items: center; font-size: 22px; color: rgba(255,255,255,0.6);">${location}</div>` : ""}
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          width: 1200,
+          height: 630,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'height' should be before 'width'.

(sort-keys-fix/sort-keys-fix)